lta/picker.py

```diff
@@ -110,16 +116,18 @@ def __init__(self, config: Dict[str, str], logger: Logger) -> None:
     # attempt to PATCH the status resource
     picker.logger.info(f"PATCH {status_url} - {status_body}")
     try:
-        session = FuturesSession()
-        # r = requests.patch(status_url, data=status_body)
-        r = await asyncio.wrap_future(session.patch(status_url, data=status_body))
-        if (r.status_code < 200) or (r.status_code > 299):
-            picker.logger.error("Unable to PATCH /status/picker with heartbeat")
-            picker.lta_ok = False
-            return picker.lta_ok
+        # TODO: Will probably refactor this into some function -- get a configured RestClient
```

Comment:
You could put it in the picker itself, or pass it in as a parameter to the function.